public/static/docs/command-reference/remote/modify.md

```diff
@@ -382,26 +382,28 @@ For more information on configuring Azure Storage connection strings, visit
 
 ## Example: Customize an S3 remote
 
-Let's first set up a _default_ S3 remote:
+Let's first set up a _default_ S3 remote.
 
-```dvc
-$ dvc remote add -d myremote s3://mybucket/storage
+> 💡 Before adding an S3 remote, be sure to
+> [Create a Bucket](https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html).
 
+```dvc
+$ dvc remote add -d myremote s3://mybucket/myproject
 Setting 'myremote' as a default remote.
 ```
 
-Modify its endpoint URL:
+Modify its access profile:
 
 ```dvc
-$ dvc remote modify myremote endpointurl https://object-storage.example.com
+$ dvc remote modify myremote profile myusername
 ```
 
-Now the config file should look like (run `cat .dvc/config`):
+Now the project config file should look like this:
```

Comment:
I would keep the place where it is located

Reply:
It's not always located there and this example doesn't start by initializing the project so it's not guaranteed to be in that location. There's several other examples of config file in other remote commands and none mention the config file location.